include/evmc/evmc.h

```diff
@@ -23,7 +23,15 @@ extern "C" {
 
 enum
 {
-    /** The EVMC ABI version number of the interface declared in this file. */
+    /**
+     * The EVMC ABI version number of the interface declared in this file.
+     *
+     * The EVMC ABI version always equals the major version number of the EVMC project.
+     * See @ref EVMC#Versioning.
+     * The Host SHOULD check if the ABI versions matches when dynamically loading EVMC VMs.
```

Comment:
`the ABI versions match` or `the ABI versions equal`, but not `the ABI versions matches`